train.py

```diff
@@ -11,41 +11,53 @@
 model_basename = config['model_basename']
 model_filename = model_basename + str(cur_iter)
 
-#Load up the training data
+# Load up the training data
 print ('Loading training data')
-#X_train is a tensor of size (num_train_examples, num_timesteps, num_frequency_dims)
-#y_train is a tensor of size (num_train_examples, num_timesteps, num_frequency_dims)
+# X_train is a tensor of size (num_train_examples, num_timesteps, num_frequency_dims)
 X_train = np.load(inputFile + '_x.npy')
+# y_train is a tensor of size (num_train_examples, num_timesteps, num_frequency_dims)
 y_train = np.load(inputFile + '_y.npy')
 print ('Finished loading training data')
 
-#Figure out how many frequencies we have in the data
+# Figure out how many frequencies we have in the data
 freq_space_dims = X_train.shape[2]
 hidden_dims = config['hidden_dimension_size']
 
-#Creates a lstm network
-model = network_utils.create_lstm_network(num_frequency_dimensions=freq_space_dims, num_hidden_dimensions=hidden_dims)
-#You could also substitute this with a RNN or GRU
-#model = network_utils.create_gru_network()
+# Creates a lstm network
+model = network_utils.create_lstm_network(
+    num_frequency_dimensions=freq_space_dims,
+    num_hidden_dimensions=hidden_dims,
+    num_recurrent_units=2
```

Comment:
Note additional hidden layers.  Perhaps also need to add some dropout between these layers?